node-api/handlers/utils/id.go

```diff
@@ -41,6 +41,9 @@ var ErrNoSlotForStateRoot = errors.New("slot not found at state root")
 func SlotFromStateID[StorageBackendT interface {
 	GetSlotByStateRoot(root common.Root) (math.Slot, error)
 }](stateID string, storage StorageBackendT) (math.Slot, error) {
+	if stateID == StateIDGenesis {
+		return 0, errors.New("genesis slot not supported")
+	}
```

Comment:
why?

Reply:
I assumed from our discussion - refactor the slotFromStateID func to alert the caller that if the input id is “genesis”. That's why added this check.  Did I miss something here ?